app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -82,11 +82,6 @@ def prepare_for_evss!
 
     save_metadata(forward_to_mas_all_claims: true) if Flipper.enabled?(:rrd_mas_all_claims_tracking) &&
                                                       !single_issue_hypertension_cfi?
-
-    if Flipper.enabled?(:rrd_mas_disability_tracking) && single_disability_eligible_for_mas?
-      save_metadata(forward_to_mas: true)
-      insert_classification_codes unless Flipper.enabled?(:rrd_mas_all_claims_notification)
```

Comment:
Should the definition of `insert_classification_codes` be removed since it's no longer called?

Reply:
Yes it should, good catch!